scripts/asciidoc2html.py

```diff
@@ -182,6 +182,7 @@ def _build_website_file(self, root, filename):
 
         asciidoc_args = ['--theme=qute', '-a toc', '-a toc-placement=manual']
         self.call(modified_src, dst, *asciidoc_args)
+        # pylint: enable=too-many-locals,too-many-statements
```

Comment:
No need for this, as pylint already only turns things off for this function and it's needed for the entire function.